apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java

```diff
@@ -207,6 +216,24 @@ INSERT INTO dataset_items (
                 ;
             """;
 
+    private static final String FIND_DATASET_ITEMS_SUMMARY_BY_DATASET_IDS = """
+                SELECT
+                    count(id) AS count,
+                    dataset_id
```

Comment:
```suggestion
                    dataset_id,
                    count(id) AS count
```

Reply:
@thiagohora Maybe I'm missing something, could you clarify please. Seems it's exactly the same, just different order?
Anyway I updated it)

Reply:
Yeah, just for readability as the aggregation is based on `dataset_id`